ddl/ddl_db_test.go

```diff
@@ -833,6 +841,17 @@ func match(c *C, row []interface{}, expected ...interface{}) {
 	}
 }
 
+func (s *testDBSuite) TestTimestamp(c *C) {
+	defer testleak.AfterTest(c)
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use " + s.schemaName)
+
+	s.tk.MustExec("create table test_timestamp (a int, b timestamp default null on update current_timestamp);")
+	s.mustExec(c, "insert into test_timestamp set a = 1")
+	rowStr := fmt.Sprintf("%v", nil)
+	s.tk.MustQuery("select b from test_timestamp").Check(testkit.Rows(rowStr))
```

Comment:
You said the test has been added in tidb-test. This PR handles the null flag, temporarily can not add this test, and I will add it at the next PR.